openedx/features/smart_referral/tests/test_views.py

```diff
@@ -0,0 +1,69 @@
+import mock
+from ddt import data, ddt
+from django.urls import reverse
+from rest_framework.status import HTTP_200_OK, HTTP_400_BAD_REQUEST, HTTP_403_FORBIDDEN
+from rest_framework.test import APITestCase
+
+from lms.djangoapps.onboarding.tests.factories import UserFactory
+
+from .factories import SmartReferralFactory
+
+
+@ddt
+class SmartReferralInvitationAPIViewTest(APITestCase):
+
+    def setUp(self):
+        super(SmartReferralInvitationAPIViewTest, self).setUp()
+        self.user = UserFactory()
+        self.user.set_password('password')
+        self.user.save()
+        self.client.login(username=self.user.username, password='password')
+
+    @mock.patch('openedx.features.smart_referral.views.task_referral_and_toolkit_emails')
+    def test_send_initial_emails_success(self, mock_task_referral_and_toolkit_emails):
+        """Successfully send referral emails to two users and current user will receive a toolkit email"""
+
+        data = """[
+            {"fist_name": "Test1", "last_name": "Referral", "contact_email": "test.referral1@example.com"},
+            {"fist_name": "Test2", "last_name": "SmartReferral", "contact_email": "test.referral2@example.com"}
+        ]"""
+        response = self.client.post(reverse('initial_referral_emails'), data=data, content_type='application/json')
+
+        contact_emails = ['test.referral1@example.com', 'test.referral2@example.com']
+        mock_task_referral_and_toolkit_emails.assert_called_once_with(contact_emails, self.user.email)
+        self.assertEqual(response.status_code, HTTP_200_OK)
+
+    @mock.patch('openedx.features.smart_referral.views.task_referral_and_toolkit_emails')
```

Comment:
Let's please use tasks.py path to mock it.

Reply:
For mocking, we use the path where the function is used instead of the path where it is created

Reply:
Thanks! I was not aware of it.